README.md

```diff
@@ -60,6 +60,17 @@ If you have issue with apache2, you can try to add `apache2.service` next to oth
 
 The log for log2ram will be written at: `/var/log/log2ram.log`
 
+| Compressor name	     | Ratio	| Compression | Decompress. |
```

Comment:
Can you explain this table ?

Reply:
Its the minimum compression ratios of several common compression algs.
Zram can use any alg that is installed and shows in /proc/crypto